README.md

```diff
@@ -1 +1,57 @@
-# nmpolicy
\ No newline at end of file
+A expressions driven declarative API for dynamic network configuration
+
+# How it works
+
+It's implemented on top of [nmstate](https://nmstate.io/), generating 
+nmstate yamls using nmpolicy and other nmstate yamls as input.
+
+This is a simple nmpolicy example to put a bridge on top of default gateway 
+nic.
+
+```yaml
+capture:
+  default-gw: routes.running.destination=="0.0.0.0/0"
+  base-iface: interfaces.name==capture.default-gw.routes.running.0.next-hop-interface
+desiredState:
+  interfaces:
+  - name: br1
+    description: Linux bridge with base interface as a port
+    type: linux-bridge
+    state: up
+    mac-address: "{{ capture.base-iface.interfaces.0.mac-address }}"
+    ipv4:
+      dhcp: true
+      enabled: true
```

Comment:
This is irrelevant to the example.

Reply:
One of the examples is the default gw refered NIC on a DHCP aware bridge.

Reply:
Added `DHCP aware` to the description.